webapp/tests/karma/unit/services/db-sync.js

```diff
@@ -257,7 +259,11 @@ describe('DBSync service', () => {
             expect(from.callCount).to.equal(3);
             done();
           });
+        }).catch((err) => {
+          window.__karma__.error(err);
         });
+      }).catch((err) => {
+        window.__karma__.error(err);
```

Comment:
I think karma will fail the test if you pass anything to the `done` function, so this could be...

```suggestion
        done(err);
```

This goes for the other tests below too...